unitTests/test_futils.cpp

```diff
@@ -141,6 +141,23 @@ TEST(AUri, parsesAndDecoreUrl)
     Uri::Decode(uri);
 }
 
+// Regression test for 
```

Comment:
It seems you forgot to add here the issue number. 

[suggestion] I would propose to rename the test case to something like:

```
TEST(AUri, parsesAndDecoreUrlWithQuestionMark)
```

I think it is more useful to have a test describing exactly what it is testing rather to point to an issue. Actually you can leave the comment pointing to the original issue (that it is not so important for me), but please, rename the test.